src/app/components/developer/developer.html

```diff
@@ -297,69 +297,52 @@ <h2 class="panel-title">{{ $ctrl.developer.name }}</h2>
+          <div  ng-if="$ctrl.hasAnyRole(['ROLE_ADMIN', 'ROLE_ONC']) && !$ctrl.isChangeRequest">
+            <div class="repeatable-item" ng-repeat="status in $ctrl.developer.statusEvents | orderBy:'-statusDateObject' track by $index">
+              <div class="flex-container">
+                <div class="flex-item ">
+                  <label for="developer-status-{{ $index }}" ><span ng-if="$first">Current </span>Developer Status <span class="text-danger">*</span></label>
+                  <select ng-model="status.status.status" required id="developer-status-{{ $index }}" name="developerStatus{{ $index }}" class="input-sm form-control">
+                    <option value="Active">Active</option>
+                    <option value="Suspended by ONC">Suspended by ONC</option>
+                    <option value="Under certification ban by ONC">Under certification ban by ONC</option>
+                  </select>
+                  <div class="text-danger" ng-if="($ctrl.showFormErrors || $ctrl.form['developerStatus' + $index].$touched) && $ctrl.form['developerStatus' + $index].$error.required">Developer Status is required</div>
+                  <div class="text-danger" ng-if="($ctrl.showFormErrors || $ctrl.form['developerStatus' + $index].$touched) && $ctrl.matchesPreviousStatus(status)">Developer Status must differ from previous Status</div>
+                </div>
+                <div class="flex-item ">
+                  <label for="status-date-{{ $index }}" >Change Date <span class="text-danger">*</span></label>
+                  <div class="input-group">
+                    <input class="input-sm form-control" type="text" uib-datepicker-popup="MM/dd/yyyy" ng-model="status.statusDateObject" is-open="status.statusDatePicker" close-text="Close" id="status-date-{{ $index }}" name="statusDate{{ $index }}" ng-model-options="{ timezone: 'UTC' }" required>
+                    <span class="input-group-btn">
+                      <button type="button" class="btn btn-link btn-sm" ng-click="status.statusDatePicker = true"><i class="fa fa-calendar"></i></button>
+                    </span>
+                  </div>
+                  <div class="text-danger" ng-if="($ctrl.showFormErrors || $ctrl.form['statusDate' + $index].$touched) && $ctrl.form['statusDate' + $index].$error.required">Change date is required</div>
+                  <div class="text-danger" ng-if="($ctrl.showFormErrors || $ctrl.form['statusDate' + $index].$touched) && $ctrl.matchesPreviousDate(status)">Only one change of status allowed per day</div>
+                </div>
+                <div class="flex-item narrowest vertical-center">
+                  <button type="button" class="btn btn-link btn-sm pull-right" ng-click="$ctrl.removeStatus($ctrl.developer.statusEvents.indexOf(status))"><i class="fa fa-close"></i><span class="sr-only">Remove status</span></button>
```

Comment:
I think the "pull-*" classes shouldn't be used with flexbox stuff. Does it work if this class is removed?

Reply:
Left over from when I was trying something.  Removed.